bodo/pandas/plan_optimizer.pyx

```diff
@@ -508,13 +509,20 @@ cdef class ConjunctionOpExpression(Expression):
         return f"ConjunctionOpExpression({self.out_schema})"
 
 
-cdef class LogicalUnaryOp(LogicalOperator):
-    cdef public object op
+cdef class UnaryOpExpression(Expression):
```

Comment:
Is this actually working? Is there a unit test?

Reply:
There is now.